### ISSUE TITLE
Feature: return 404 on /api/inlets/default and /api/messages in current version

### ISSUE THREAD

**Issue by benchonaut** [muety/telepush#52]
(update:

Everything works as expected, 
version 3 ( named telepush) removed the `/api/messages` endpoint,
but rejects it with `405` == "method not allowed", which is technically correct ( more or less ).
Unfortunately this isn't interpreted as failure by `curl --fail`

scripts used were changed to try v3 (`/api/messages/YOUR-Tic-Tac-Token`) first , 
but some clients ( not physically reachable ) might still try to use the old method 
which does not work unless a frontend proxy is used like this ( nginx example)
```
    location ~ ^/api/messages$ {
         return 404;
         access_log /dev/null;
    }
```

so the request is: 
Please return `404` on `/api/messages` and `/api/inlets/default`  ( exact path),
 which would be technically correct as well ( endpoint does not exist ) and make "curl --fail" work

Pull Request here: https://github.com/muety/telepush/pull/54

Thank You

---
OLD TEXT:

```

(Original Text 2)

* the router seems to fail on 
* * `/api/messages/YOUR-Tic-Tac-Token`
* * `/api/inlets/default/YOUR-Tic-Tac-Token`


( solution/problem: 
in the below "original ticket" text , the endpoint `/api/messages` was blocked by nginx
to speed up the detection of v2/v3 telepush,

however , the "current" docker image does not seem to accept the urls including tokens..

so if you need to send something , the "recipient_token" has to be in json ,
then both "non-tokenized" endpoints `/api/inlets/default` and `api/messages` will work


)


(Original Text 1)
## Situation:

* had the thingy running in longpoll mode , logs filled up with "destination unreachable" and other warnings
* in a specific chat "sending links" was forbidden for a certain time
* the script used to send the messages is https://gitlab.com/the-foundation/pam-notify-telegram/
* thingy did not send to specific chat anymore , but to all the others
* thingy did still respond to "/start" command
* started thingy in webhook mode 
* manually set webhook ( will document in another ticket)
* created a new token ( for the not-working chat )
* new token works
* as many clients ( some physically hard to reach ) use the "damaged" token , some approaches were tried to make the old location work again: 

* * 301/302 redirect  /api/messages/old-token -> /api/messages/new-token : client follows to new location , sends exact same byte size , message is not sent
* * nginx rewrite with extra location /api/messages/old-token /api/messages/new-token ( verified with extra host/port and rewrite_log)


## Your Question was ?
is there any chance to 

* implement a "token re-write" easily 
* debug the internal storage 
* "reset" any error marker for that chat ( if there is smth like that )
*  delete the old token and change a new one
* make it accept 301 requests 
* or "anything that works" ?

```

**Comment by benchonaut** [comment]
( Thank you  a lot )

**Comment by benchonaut** [comment]
( Thank you a lot for https://github.com/muety/telepush/commit/676dc3e04c927db9aa8927533d04e0ebe1ce03dc  ;) )
